src/Datadog.Trace/Agent/Api.cs

```diff
@@ -195,6 +146,63 @@ private static HashSet<ulong> GetUniqueTraceIds(Span[][] traces)
             return uniqueTraceIds;
         }
 
+        private async Task<bool> SendTracesAsync(Span[][] traces, IApiRequest request)
+        {
+            IApiResponse response = null;
+
+            try
+            {
+                try
+                {
+                    _statsd?.AppendIncrementCount(TracerMetricNames.Api.Requests);
+                    response = await request.PostAsync(traces, _formatterResolver).ConfigureAwait(false);
+                }
+                catch
+                {
+                    // count the exceptions thrown by the HttpClient,
+                    // not responses with 5xx status codes
+                    // (which cause EnsureSuccessStatusCode() to throw below)
```

Comment:
Nit: This comment is outdated, we can't call `EnsureSuccessStatusCode()` anymore.

Reply:
Actually, I think `WebRequest` throws for 5xx response codes, so we're probably not counting the API errors correctly anymore, but that's out of scope for this PR.